client/src/js/references/components/Detail/Targets/__tests__/Form.test.js

```diff
@@ -0,0 +1,39 @@
+import { Checkbox, Input } from "../../../../../base";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Input' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5211723)